sbe-tool/src/test/resources/extension-schema.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<sbe:messageSchema xmlns:sbe="http://fixprotocol.io/2016/sbe"
+                   package="code.generation.test"
+                   id="1"
+                   version="4"
+                   description="Unit Test">
+    <types>
+        <composite name="messageHeader" description="Message identifiers and length of message root">
+            <type name="blockLength" primitiveType="uint16"/>
+            <type name="templateId" primitiveType="uint16"/>
+            <type name="schemaId" primitiveType="uint16"/>
+            <type name="version" primitiveType="uint16"/>
+            <type name="headerLength" primitiveType="uint16"/>
+
+            <type name="extraField" primitiveType="int32" sinceVersion="5"/>
```

Comment:
The [spec](https://github.com/FIXTradingCommunity/fix-simple-binary-encoding/blob/master/v1-0-STANDARD/doc/05SchemaExtensionMechanism.md) states "Message header encoding cannot change."